src/wrapped/wrappedlibtiff5_private.h

```diff
@@ -0,0 +1,43 @@
+#if !(defined(GO) && defined(GOM) && defined(GO2) && defined(DATA))
+#error Meh....
+#endif
+
+//GO(TIFFVGetFieldDefaulted,
+// GO(TIFFVGetField, iFpWA)
+GO(TIFFCleanup, vFp)
+GO(TIFFFreeDirectory, vFp)
+GO(TIFFFlush, iFp)
+GO(TIFFClose, vFp)
+GO(TIFFFindCODEC, pFW)
+GO(TIFFIsCODECConfigured, iFW)
+GO(TIFFInitDumpMode, iFpi)
+GO(TIFFInitLZW, iFpi)
+//GO(TIFFInitPackBits
+GO(TIFFInitThunderScan, iFpi)
+GO(TIFFInitNeXT, iFpi)
+GO(TIFFInitJPEG, iFpi)
+//GO(TIFFInitOJPEG
+GO(TIFFInitCCITTRLE, iFpi)
+GO(TIFFInitCCITTRLEW, iFpi)
+GO(TIFFInitCCITTFax3, iFpi)
+//GO(TIFFInitCCITTFax4
+GO(TIFFInitJBIG, iFpi)
+GO(TIFFInitZIP, iFpi)
+//GO(TIFFInitPixarLog
+//GO(TIFFInitSGILog
+GO(TIFFInitLZMA, iFpi)
+GO(TIFFInitZSTD, iFpi)
+GO(TIFFInitWebP, iFpi)
+// GO(TIFFCIELabToXYZ, vFpuiippp)
+GO(TIFFXYZToRGB, iFpppp)
```

Comment:
void TIFFXYZToRGB(TIFFCIELabToRGB *cielab, float X, float Y, float Z, uint32_t *R, uint32_t *G, uint32_t *B)

I’m on my phone, so I randomly picked a function to check, and it seems badly wrong?

Reply:
Ok, I concur, this function has a bad signature.

@kaixindeken how are you making the function signatures? Should I revert all your PR?

Reply:
(man page from here: https://www.manpagez.com/man/3/TIFFXYZToRGB/ )

Reply:
my apologis, i'll fix it right now. please do not revert this pr. i'll create a new pr to fix it all.